src/main/java/ch/hevs/cloudio/endpoint/CloudioFactoryNode.java

```diff
@@ -0,0 +1,13 @@
+package ch.hevs.cloudio.endpoint;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
Using
```
@JsonAnySetter
private Map<String, Object> properties = new LinkedHashMap<>();
```
you can automate deserialisation of the additional properties to a map.